diff --git a/dist/index.js b/dist/index.js
index 330f4809a7d10f19f66a832d922ffd8a4b61876e..927e49ba9044f88d7ecc2a25d7ab1b5136d60701 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1185,7 +1185,9 @@ function buildSetup(meta) {
                 map
               ]);
             } else {
-              map = result.map;
+              if(plugin.name !== 'unplugin-macros') {
+                map = result.map;
+              }
             }
           }
           if (code) {
